astrokat/noisediode.py

```diff
@@ -219,20 +240,22 @@ def pattern(kat,  # kat subarray object
     # - add a default lead time to ensure enough time for all digitisers
     #   to be set up
     timestamp = np.ceil(time.time() + lead_time)
-    msg = ('Set noise diode pattern to activate at {}, '
-           'with {} sec lead time'
+    msg = ('Request: Set noise diode pattern to activate at {} '
+           '(includes {} sec lead time)'
            .format(timestamp,
                    lead_time))
     user_logger.warning(msg)
 
     if nd_antennas == 'all':
         # Noise Diodes are triggered on all antennas in array simultaneously
         # add a second to ensure all digitisers set at the same time
-        replies = kat.ants.req.dig_noise_source(timestamp, on_fraction, cycle_length)
+        replies = kat.ants.req.dig_noise_source(timestamp,
+                                                on_fraction,
+                                                cycle_length)
         if not kat.dry_run:
-            _katcp_reply_to_log_(replies)
+            timestamp = _katcp_reply_to_log_(replies)
         else:
-            msg = ('Set all noise diodes with timestamp {} ({})'
+            msg = ('Dry-run: Set all noise diodes with timestamp {} ({})'
                    .format(int(timestamp),
```

Comment:
In `on` and `off`, the dry-run cases are logging with `ceil` (but probably shouldn't use `int` or `ceil`)